### ISSUE TITLE
Arguments to -t reversed?

### ISSUE THREAD

**Issue by aklaing** [jonls/redshift#371]
I use redshift on Ubuntu 16.04.  The man page for redshift says:

```
       -t DAY:NIGHT
              Color temperature to set at daytime/night
```

When I run:

`redshift -l 42:71 -b 0.7 -t 5500:1800`

it seems redshift is broken.  I expect it to use color temperature 5500 (which is brighter) during the day, and 1800 (which is redder/yellower) at night.  It does the exact opposite.  So it seems not to work at night, and then during the day it becomes yellower/reddish.  When I reverse the order of the color temperatures as follows:

`redshift -l 42:71 -b 0.7 -t 1800:5500`

Then it does what I expect.  So it seems the man page is wrong, or else the arguments to `-t` have been reversed.  

My machine says "Using method 'randr'", in case that is helpful.
Otherwise redshift is a fantastic tool!  Thank you.

**Comment by EvertonMelo** [comment]
Hello, 

you try use change .conf local?
the path is: ~/.config/redshift.conf

use the content:
```

; Global settings for redshift
[redshift]
; Set the day and night screen temperatures
temp-day=5700
temp-night=3500

; Enable/Disable a smooth transition between day and night
; 0 will cause a direct change from day to night screen temperature.
; 1 will gradually increase or decrease the screen temperature.
transition=1

; Set the screen brightness. Default is 1.0.
;brightness=0.9
; It is also possible to use different settings for day and night
; since version 1.8.
;brightness-day=0.7
;brightness-night=0.4
; Set the screen gamma (for all colors, or each color channel
; individually)
gamma=0.8
;gamma=0.8:0.7:0.8
; This can also be set individually for day and night since
; version 1.10.
;gamma-day=0.8:0.7:0.8
;gamma-night=0.6

; Set the location-provider: 'geoclue', 'geoclue2', 'manual'
; type 'redshift -l list' to see possible values.
; The location provider settings are in a different section.
location-provider=manual

; Set the adjustment-method: 'randr', 'vidmode'
; type 'redshift -m list' to see all possible values.
; 'randr' is the preferred method, 'vidmode' is an older API.
; but works in some cases when 'randr' does not.
; The adjustment method settings are in a different section.
adjustment-method=randr

; Configuration of the location-provider:
; type 'redshift -l PROVIDER:help' to see the settings.
; ex: 'redshift -l manual:help'
; Keep in mind that longitudes west of Greenwich (e.g. the Americas)
; are negative numbers.
[manual]
lat=48.1
lon=11.6

; Configuration of the adjustment-method
; type 'redshift -m METHOD:help' to see the settings.
; ex: 'redshift -m randr:help'
; In this example, randr is configured to adjust screen 1.
; Note that the numbering starts from 0, so this is actually the
; second screen. If this option is not specified, Redshift will try
; to adjust _all_ screens.
[randr]
screen=1
```

aftert try:
`
redshift -l 55.7:12.6 -t 5700:3600 -g 0.8 -m randr -v`

**Comment by EvertonMelo** [comment]
But saying it:

"...Certain video drivers do not support adjustable gamma ramps. In some cases Redshift will fail with an error message, but other drivers silently ignore adjustments to the gamma ramp..."
fonte: http://jonls.dk/redshift/

what is the DRIVER which you are using?

**Comment by aklaing** [comment]
I'm using the nouveau driver:

  *-display
       description: VGA compatible controller
       product: GK106M [GeForce GTX 765M]
       vendor: NVIDIA Corporation
       physical id: 0
       bus info: pci@0000:01:00.0
       version: a1
       width: 64 bits
       clock: 33MHz
       capabilities: pm msi pciexpress vga_controller bus_master cap_list
rom
       configuration: driver=nouveau latency=0
       resources: irq:28 memory:dc000000-dcffffff memory:b0000000-bfffffff
memory:c0000000-c1ffffff ioport:e000(size=128) memory:dd000000-dd07ffff

**Comment by aklaing** [comment]
I tried using your config file.  I had to change Screen=1 to Screen=0.
Apart from that I observed the behavior I previously described.  That is,
if I use

-t <low-temp>:<high-temp>

during the day, then the <high-temp> is implemented.  OTOH if I use

-t <high-temp>:<low-temp>

during the day, then the <low-temp> is implemented.

Thanks

**Comment by aklaing** [comment]
Excuse me if you get this twice: I am using reply-all this time to make
sure everyone gets the response.
I'm using the nouveau driver:

  *-display
       description: VGA compatible controller
       product: GK106M [GeForce GTX 765M]
       vendor: NVIDIA Corporation
       physical id: 0
       bus info: pci@0000:01:00.0
       version: a1
       width: 64 bits
       clock: 33MHz
       capabilities: pm msi pciexpress vga_controller bus_master cap_list
rom
       configuration: driver=nouveau latency=0
       resources: irq:28 memory:dc000000-dcffffff memory:b0000000-bfffffff
memory:c0000000-c1ffffff ioport:e000(size=128) memory:dd000000-dd07ffff

**Comment by aklaing** [comment]
Pardon the duplicate -- I am using reply-all now, to make sure everyone
gets the response:

I tried using your config file.  I had to change Screen=1 to Screen=0.
Apart from that I observed the behavior I previously described.  That is,
if I use

-t <low-temp>:<high-temp>

during the day, then the <high-temp> is implemented.  OTOH if I use

-t <high-temp>:<low-temp>

during the day, then the <low-temp> is implemented.

Thanks
